package.json

```diff
@@ -2,6 +2,9 @@
   "name": "gateway",
   "version": "0.0.0",
   "private": true,
+  "overrides": {
+    "@types/node": "20.8.3"
```

Comment:
Why do we need this?

Reply:
To avoid conflicts between `undici types` and `wrangler types`, I restricted the type resolver to version `20.8.3`. The node community created types similar to those provided by Cloudflare wrangler, which are loaded transitively into the development environment, causing errors throughout the codebase. These problematic types were introduced after node version `20.8.3`. By overriding the type resolver to strictly use `20.8.3`, errors are prevented, and the correct methods and parameters are suggested by Intelli Sense.